c/classmate/classmate_8.4.sh

```diff
@@ -0,0 +1,49 @@
+#----------------------------------------------------------------------------
+#
+# Package		: classmate
+# Version		: 1.3.4, 1.5.1
+# Source repo		: https://github.com/FasterXML/java-classmate
+# Tested on		: RHEL 8.4
+# Script License	: Apache License Version 2.0
+# Maintainer		: Vikas Gupta <vikas.gupta8@ibm.com>
```

Comment:
Please update the header as per the templates.

Reply:
Updated latest template.